spec/models/application_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Application, type: :model do
+    describe "Relationships" do 
+        it {should have_many :pet_applications}
```

Comment:
Make sure you add this relationship test as well
`it { should have_many(:pets).through(:pet_applications) } `